feature.go

```diff
@@ -0,0 +1,11 @@
+package xss
+
+import (
+	"io/ioutil"
+	"net/http"
+)
+
+func Test() {
+	r, _ := http.Get("")
+	ioutil.ReadAll(r.Body)
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error return value of `ioutil.ReadAll` is not checked (errcheck)